src/nsidc/metgen/cli.py

```diff
@@ -33,6 +33,16 @@ def info(config_filename):
     metgen.init_logging(configuration)
     configuration.show()
 
+@cli.command()
+@click.option('-c', '--config', 'config_filename', help='Path to configuration file', required=True)
+@click.option('-t', '--type', 'content_type', help='JSON content type', default='cnm', show_default=True)
+def validate(config_filename, content_type):
+    """Validates the contents of local JSON files."""
+    click.echo(metgen.banner())
+    configuration = config.configuration(config.config_parser_factory(config_filename), {})
+    metgen.init_logging(configuration)
+    metgen.validate(configuration, content_type)
```

Comment:
May want to insert a validation of the config file? I'm not sure it's necessary, because there could be lots of stuff in the config file that we don't care about when validating the json. https://github.com/nsidc/granule-metgen/blob/main/src/nsidc/metgen/cli.py#L54

Reply:
My take on the story is that we're focused on validating JSON output in the issue-41 feature, not other things that do in fact need to be validated. I'm inclined to stick with the current setup. What I feel would be better, config-file-wise, is to add the equivalent of`config.validate(configuration)` to the CLI `init` processing so that the new `.ini` file is immediately checked for errors.